.eslintignore

```diff
@@ -0,0 +1,17 @@
+# Files and folders related to build/test tools
+/build
+/node_modules
+/tests
+/vendor
+/tools
+/Gruntfile.js
+/webpack.config.js
```

Comment:
I think these files should be included in the ESLint checks, any particular reason you chose to exclude them @whyisjake ?

Reply:
Updated in the patches/commit.